common/src/main/java/whocraft/tardis_refined/common/blockentity/console/GlobalConsoleBlockEntity.java

```diff
@@ -83,7 +84,7 @@ public void load(CompoundTag tag) {
 
         super.load(tag);
 
-        spawnControlEntities();
+//        spawnControlEntities();
```

Comment:
Why is this commented?

Reply:
Temporarily commented out for debugging, will resolve later